content/zh/docs/concepts/scheduling/kube-scheduler.md

```diff
@@ -311,6 +311,10 @@ kube-scheduler 有一系列的默认调度策略。
 {{% /capture %}}
 {{% capture whatsnext %}}
 * 阅读关于 [调度器性能调优](/zh/docs/concepts/scheduling/scheduler-perf-tuning/)
+
+* 进一步了解 [控制器]({{< indoclink pagepath="/docs/concepts/architecture/" >}})
```

Comment:
This generates a hyperlink to `/zh/docs/concepts/architecture` (note: no trailing `/`). Is it easy to preserve the trailing `/`?

Reply:
I have been updated the shortcode `doclink` implement, it should works as expect right now, although the impl seems little uggly.